aspnet/fundamentals/logging.rst

```diff
@@ -73,23 +96,37 @@ Exception error
 
 .. note:: Some loggers, such as the built-in ``ConsoleLogger`` used in this article, will ignore the ``eventId`` parameter. If you need to display it, you can include it in the message string. This is done in the following sample so you can easily see the eventId associated with each message, but in practice you would not typically include it in the log message.
 
+In the ``TodoController`` example, event id constants are defined for each event, and log statements are configured at the appropriate verbosity level based on the success of the operation. In this case, successful operations log as ``Information`` and not found results are logged as ``Warning`` (error handling is not shown).
+
+.. literalinclude:: logging/sample/src/TodoApi/Controllers/TodoController.cs
+	:language: c#
+	:linenos:
+	:lines: 24-43
+	:dedent: 4
+	:emphasize-lines: 4,12,16
+
+.. note:: It is recommended that you perform application logging at the level of your application and its APIs, not at the level of the framework. The framework already has logging built in which can be enabled simply by setting the appropriate logging verbosity level.
+
+Logging in Middleware
```

Comment:
Now that we have the Todo app sample I think you can remove this section and the middleware sample. The framework provides this level of logging without adding any additional logging middleware. We don't want to mislead folks into thinking they need to add this level of logging themselves.